docs/usage/index.rst

```diff
@@ -9,5 +9,6 @@ This section covers the usage of edgegraph as a consumer of the module.
    :glob:
 
    *
+   algos/index
```

Comment:
Check this shows correctly on RTD -- had issues finding the page on local doc build.